services/secret-service/doc/openapi.yaml

```diff
@@ -331,9 +436,45 @@ components:
         version:
           type: string
     MutableAuthClient:
+      required:
+        - name
+        - owners
```

Comment:
The property `owners` should belong to `AuthClient`, not `MutableAuthClient`

Reply:
Why not? Why do you have `owners` then in you initial MutableSecret? And how should owners be modified, if not through e.g. a patch request?

Reply:
You are right. My fault.